sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-nacos/src/main/java/org/apache/shardingsphere/orchestration/center/instance/NacosConfigInstance.java

```diff
@@ -108,16 +106,12 @@ private String getDirectly(final String key) {
      */
     @Override
     public void persist(final String key, final String value) {
```

Comment:
Whether should check update in schemas.

Reply:
I think the check update should for `overwrite`, if user set is true, we should overwrite value directly, if false, we do nothing.